### ISSUE TITLE
[Oversight] minetweaker.item should say crafttweaker.item

### ISSUE THREAD

**Issue by MysticAura** [jaredlll08/CraftTweaker-Documentation#3]
I couldn't figure out why my script was saying my IItemStack import was null, so I started digging through your source and figured it out. In the docs it says minetweaker.item but it should say crafttweaker.item

[image for reference](https://i.gyazo.com/963a1be829083e4e509d08b97bb99a3a.png)

For the record, I love the new docs. They're so much nicer than the old ones.

**Comment by kindlich** [comment]
Yeah, kinda my fault because I used 1.11 when creating most scripts and back then it was minetweaker instead of crafttweaker.

I think jared already fixed this
